clients/js/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/*
+.DS_Store
+npm-debug.log
```

Comment:
File necessary to avoid pushing large `node_modules` folder inside the repository.